python/ray/autoscaler/command_runner.py

```diff
@@ -613,37 +621,39 @@ def run(
             with_output=with_output,
             ssh_options_override_ssh_key=ssh_options_override_ssh_key)
 
-    def run_rsync_up(self, source, target):
-        # TODO(ilr) Expose this to before NodeUpdater::sync_file_mounts
-        protected_path = target
-        if target.find("/root") == 0:
-            target = target.replace("/root", "/tmp/root")
+    def run_rsync_up(self, source, target, options=None):
```

Comment:
For code where options=None is actually faulty (i.e., it is assumed to be a dict), you will need to resolve it inline:
```
def run_rsync_up(..., options=None):
    options = options or {}

Reply:
otherwise we're basically assuming that options is always mandatory right?

Reply:
Oh! Yep, totally makes sense.